requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.4.18
+Django==1.7.7
```

Comment:
I did try upgrading django-allauth but it seemed to break more things. What would be the strategy for upgrading other django dependencies?